js/app.js

```diff
@@ -1 +1,165 @@
-console.log('Hello frontend');
+document.addEventListener('DOMContentLoaded', function(){
+
+//set the value for the first turn
+var turn = "X";
+//function to alternate between x and o
+function switchTurn() {
+  // if(checkForWin()) {
+  //   document.getElementById("gameOverMessage").textContent = "Win!"
+  // }
+  if (turn == "X") {
+    turn = "O";
+  } else {
+    turn = "X";
+  }
+}
+
+// //ATTEMPT TO CREATE A LOOP FOR ALL CLICKABLE CELLS
+// var clickBox = document.querySelectorAll('.cell');
+//   console.log(clickBox);
+//   clickBox.addEventListener('click', function(e){
+//     var clickBoxNow = document.querySelectorAll('.cell');
+//     for (var i = 0; i < clickBoxNow.length; i++) {
+//       clickBoxNow[i].textContent = turn;
+//       switchTurn();
+//     }
+//   e.preventDefault();
```

Comment:
Don't need e.preventDefault() here... however, you have access to the clicked box in this function with e.srcElement. Take a look at all the information you have available to you by logging e.srcElement to the console.